packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -166,11 +168,12 @@ const typeDefs = gql\`${startSchema}\`
     }
   }
 
-  public use({ getHttp, path }: RegistrationOptions) {
+  public use({ getHttp, path, cache }: RegistrationOptions) {
```

Comment:
I'm not sure I get what this method is for. It has a pretty vague name and no docs. It's for use by registerServer implementations?